packages/grafana-alerting/src/grafana/api/v0alpha1/api.ts

```diff
@@ -2,16 +2,17 @@ import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
 
 import { getAPIBaseURL, getAPIReducerPath } from '../util';
 
-const VERSION = 'v0alpha1';
-const GROUP = 'notifications.alerting.grafana.app';
+import { GROUP, VERSION } from './const';
 
 const baseUrl = getAPIBaseURL(GROUP, VERSION);
 const reducerPath = getAPIReducerPath(GROUP, VERSION);
 
 export const api = createApi({
   reducerPath,
   baseQuery: fetchBaseQuery({
-    baseUrl,
+    // Set URL correctly so MSW can intercept requests
+    // https://mswjs.io/docs/runbook#rtk-query-requests-are-not-intercepted
+    baseUrl: new URL(baseUrl, globalThis.location.origin).href,
```

Comment:
not related to the goal to this PR, but, I think we should add here the `tagTypes` used in the `api.gen.ts`, right?

Reply:
So the client we export is actually the generated one, which does include the tags (it calls `enhanceEndpoints()`) so this file doesn't need the tag definitions – users of this package don't have access to this internal api file :)